### ISSUE TITLE
Umbau des Taktgebers vom Microcontroller zu Kurvenscheibe

### ISSUE THREAD

**Issue by auenkind** [auenkind/YACoVV#2]
Das Microcontroller / Servo Prinzip des Prototypen soll mit einem Kurvenscheiben System ersetzt werden.

**Comment by Pixtxa** [comment]
[Hinweis zu Servo-Motoren: Ich hatte mal in einem Projekt einen billigen eingesetzt. Nach wenigen Stunden waren die Plastik-Zahnräder so weit verschlissen, dass der Servo stark quietschte und für mein Projekt dann auch zu wenig Kraft hatte. Ist also sehr sinnvoll, hier etwas anderes einzusetzen.]

**Comment by auenkind** [comment]
> [Hinweis zu Servo-Motoren: Ich hatte mal in einem Projekt einen billigen eingesetzt. Nach wenigen Stunden waren die Plastik-Zahnräder so weit verschlissen, dass der Servo stark quietschte und für mein Projekt dann auch zu wenig Kraft hatte. Ist also sehr sinnvoll, hier etwas anderes einzusetzen.]

Ja, Servos halten die ca. 21000 Hübe / 24 Std. auf keinen Fall aus. Deshalb dieser Task der zum Ziel hat den Microcontroller und die Servos durch einen Scheibenwischermotor mit Kurvenscheiben zu ersetzen.
